wfl/pipeline/__init__.py

```diff
@@ -1,2 +1,5 @@
-from .base import iterable_loop
+from .base import iterable_loop, iloop, iloop_docstring_pre, iloop_docstring_post
 assert iterable_loop
+assert iloop
+assert iloop_docstring_pre
```

Comment:
Why the asserts?

Reply:
I don't know.  They were there for `iterable_loop`, so I assumed someone added them for a reason.  Let me see if I can figure out why.

Reply:
It was in a commit labeled "Lots of flake8 corrections in wfl/qurefun and wfl/pipeline", so I suspect it gave a warning about an unused import or something like that.